apiserver/facades/client/storage/storage.go

```diff
@@ -785,8 +785,12 @@ func (a *APIv3) destroy(args params.DestroyStorage) (params.ErrorResults, error)
 			result[i].Error = common.ServerError(err)
 			continue
 		}
+		destroy := a.storage.DestroyStorageInstance
```

Comment:
Done. Also, I'll be renaming the method to Remove in the next PR, after discussion with Christian.